cou/apps/auxiliary.py

```diff
@@ -370,6 +375,41 @@ def _get_change_require_osd_release_step(self) -> PreUpgradeStep:
             coro=set_require_osd_release_option(ceph_mon_unit.name, self.model),
         )
 
+    def _get_wait_until_noout_set_step(self) -> PreUpgradeStep:
+        """Get wait until noout is set step.
+
+        :return: Step to wait until noout is set.
+        :rtype: PreUpgradeStep
+        """
+
+        async def _prompted_verification() -> None:
```

Comment:
Why this is nested function inside another function?

Reply:
it can move out as a stand alone function; it was very small at the beginning when testing

Reply:
Removed relevant code